index.html

```diff
@@ -9,76 +9,259 @@
+                  <li class="card_btns"><card_btns_font class="card_btns_font">Ruby</card_btns_font></li>
+                  </ul>
+            </li>
+          </div>
+
+          <div id="card_works">
+            <div id="action">
+              <img src="./assets/Mask Group.png" alt="Professional art data">
+            </div>
+            <a href="https:github.com"><div id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
+            </div></a>
+
+            <li id="card_data">
+              <div id="card_title"><p>Profesional Art Printing Data</p></div>
+              <div id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></div>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">bootstrap</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">Ruby</card_btns_font></li>
+                  </ul>
+            </li>
+          </div>
+
+          <div id="card_works">
+            <div id="action">
+              <img src="./assets/Mask Group.png" alt="Professional art data">
+            </div>
+            <a href="https:github.com"><div id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
+            </div></a>
+
+            <li id="card_data">
+              <div id="card_title"><p>Profesional Art Printing Data</p></div>
+              <div id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></div>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">bootstrap</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">Ruby</card_btns_font></li>
+                  </ul>
+            </li>
+          </div>
+
+          <div id="card_works">
+            <div id="action">
+              <img src="./assets/Mask Group.png" alt="Professional art data">
+            </div>
+            <a href="https:github.com"><div id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
+            </div></a>
+
+            <li id="card_data">
+              <div id="card_title"><p>Profesional Art Printing Data</p></div>
+              <div id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></div>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
```

Comment:
Kindly, in previous reviews I told you only use valid HTML labels, and turn them into correct or divs, I attached documentation about HTML and its correct uses. (Check all your code to remove them all) https://www.tutorialspoint.com/html5/html5_tags.htm

Reply:
can you clarify what the wrong HTML label here is?